src/main/java/carracing/domain/Car.java

```diff
@@ -28,4 +38,10 @@ public int move(int cmd) {
         }
         return this.location;
     }
+
+    public void checkNameLength(String name) {
+        if (name.length() > NAME_LIMIT_LENGTH) {
+            throw new IllegalArgumentException("자동차 이름은 5 이하여야 합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("자동차 이름은 " + NAME_LIMIT_LENGTH + " 이하여야 합니다.");
```

와 같이 작성한다면 조건이 변경 됐을 때 메시지를 수정하지 않아도 되겠네요 :)